packages/@aws-cdk/aws-batch/lib/compute-environment.ts

```diff
@@ -565,6 +559,25 @@ export class ComputeEnvironment extends Resource implements IComputeEnvironment
     return securityGroups.map((group: ec2.ISecurityGroup) => group.securityGroupId);
   }
 
+  private buildComputeEnvironmentDefaultServiceRoleArn(isFargate:boolean): string {
+    if (isFargate) {
+      return Stack.of(this).formatArn({
+        service: 'iam',
+        region: '',
+        resource: 'role',
+        arnFormat: ArnFormat.SLASH_RESOURCE_NAME,
+        resourceName: 'aws-service-role/batch.amazonaws.com/AWSServiceRoleForBatch',
```

Comment:
This seems to be making an assumption that the user has previously gone through and AWS Batch getting started experience, (either in the console, SDK, or CLI) and the AWSServiceRoleForBatch Role exists in their account. Am I interpreting that correctly? I think this would fail if that role does not exist in the environment being deployed to. 

Instead of getting the role ARN from the account, we should create a new role and attach the AWS managed policy that goes with it, which I am pretty sure is called "aws-service-role/BatchServiceRolePolicy". (I know this just from doing the Batch getting started experience in the console and then checking what policy got attached to the AWSServiceRoleForBatch Role.)